pytorch/buildspec.yml

```diff
@@ -23,6 +23,16 @@
       example_mnist_file:
         source: docker/build_artifacts/mnist.py
         target: mnist.py
+    inference_context: &INFERENCE_CONTEXT
+      torchserve-entrypoint:
+        source: docker/build_artifacts/torchserve-entrypoint.py
+        target: torchserve-entrypoint.py
+      config:
+        source: docker/build_artifacts/config.properties
+        target: config.properties
+      deep_learning_container:
+        source: docker/build_artifacts/deep_learning_container.py
+        target: deep_learning_container.py
 
   images:
```

Comment:
Need to add BuildCPUPTInferencePy3DockerImage and BuildGPUPTInferencePy3DockerImage as well to build Inference Images.